### DISCUSSION TITLE
possible to arrange checklist horizontally instead of its default vertical position?

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#922] [Q&A] [answered]
Is it possible to take a checklist and expand the options horizontally instead of creating a vertical list? If not maybe some parameter for components like this would be nice!
EX:

current:
x
y
z
ideal:
x y z

**Comment by mturoci** [comment, marked as answer]
This can be achieved via `ui.inline` and individual checkboxes. However, the downsides would be:
* No select / deselect all out of the box.
* The output data wouldn't be an array of values like in checklist (q.args.checklist = [checkbox1, checkbox2], but rather individual `q.args` (q.args.checkbox1, q.args.checkbox2 ...).

If these limitations are not acceptable though, introducing `inline` flag to checklist is the only option I guess - cc @lo5

**Reply by Alex-Wenner-FHR** [reply]
Thanks for the solution!

**Reply by lo5** [reply]
Yes, adding an `inline` attribute to the checklist would be better.

**Reply by Alex-Wenner-FHR** [reply]
Is this currently possible? I do agree this would be a better solution. Also I notice that the args of the checklist arrange themselves in alphabetical order... in my use case it would be better if these args were arranged in the order they are clicked. Just a thought!
